README.md

```diff
@@ -2,6 +2,32 @@
 
 This is a work in progress implementation of the IPFS integration of [Chlu](https://chlu.io).
 
+See [CONTRIBUTING.md](https://github.com/ChluNetwork/chlu-ipfs-support/blob/master/CONTRIBUTING.md) to hack on the code.
+
 ## Usage
 
-See [CONTRIBUTING.md](https://github.com/ChluNetwork/chlu-ipfs-support/blob/master/CONTRIBUTING.md)
\ No newline at end of file
+__Note__: this has only been tested in Node but is supposed to work in the browser using Webpack. Please check out the js-ipfs information about running in the browser through Webpack for additional information.
+
+This module has not been release to NPM yet. You can however install it and import/require it, but it's still WIP. Check out [index.js](https://github.com/ChluNetwork/chlu-ipfs-support/blob/master/src/index.js) to see the available API calls, and the tests for examples.
+
+You can install this module globally and use `chlu-service-node` to run a Chlu Service Node.
```

Comment:
It can also be installed locally: in any folder, run `npm install chlu-ipfs-support`. The binary should be available under `./node_modules/chlu-ipfs-support/bin/` like any other npm module.

This binary just starts ChluIPFS and its service node routines for now, there are still a lot of improvements to make and tests to run